src/main/kotlin/org/opensearch/commons/utils/ValidationHelpers.kt

```diff
@@ -75,14 +75,22 @@ fun validateIamRoleArn(roleArn: String) {
     require(roleArnRegex.matcher(roleArn).find()) { "Invalid AWS role ARN: $roleArn " }
 }
 
-fun isValidName(name: String): Boolean {
+fun isValidQueryName(name: String): Boolean {
```

Comment:
Could you add a short doc comment that explains this function validates the doc level monitor DocLevelQuery name specifically? We use the term "query" in many places, so a quick comment can help prevent confusion.
https://github.com/opensearch-project/common-utils/blob/main/src/main/kotlin/org/opensearch/commons/alerting/model/DocLevelQuery.kt#L16